common/components/add-another/template.njk

```diff
@@ -42,7 +42,7 @@
     {% endif %}
   {% endfor %}
 
-  <div class="app-add-another__add-item {{ 'app-border-top-1 govuk-!-padding-top-5' if items.length > 0 }}">
+  <div class="app-add-another__add-item {{ 'app-border-top-1 govuk-!-padding-top-5 govuk-!-margin-top-4' if items.length > 0 }}">
```

Comment:
Does this mean there is no test coverage for property removal?

Reply:
There are tests around the `add-another` component here `common/components/add-another/template.test.js`. But not directly testing that it has any styles associated to the component itself it seems. @matt-crowson-moj

Reply:
The property removal uses this component.